server/data/UseOfForceReport.ts

```diff
@@ -14,6 +14,7 @@ export type UseOfForceDetails = {
   pavaUsed: boolean
   guidingHold: boolean
   guidingHoldOfficersInvolved: number
+  escortingHold?: boolean
```

Comment:
is this optional ?

Reply:
I think so, given previously created reports won't have a value for it